packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts

```diff
@@ -323,31 +320,31 @@ test.describe('PieCookieBanner - Component tests', () => {
             );
 
             // Act
-            const acceptAllButton = component.locator('[data-test-id="actions-accept-all"]');
-            const necessaryOnlyButton = component.locator('[data-test-id="actions-necessary-only"]');
+            const acceptAllButton = await pieCookieBannerComponent.getAcceptAllAttribute('variant');
```

Comment:
Nit: I would rename these variables to be `acceptAllButtonVariant` and `necessaryOnlyButtonVariant` to make the expectations read a bit more smoothly